src/graphics/program-lib/standard.js

```diff
@@ -879,6 +879,10 @@ pc.programlib.standard = {
         var codeBegin = code;
         code = "";
 
+        if (options.useSpecularOverAlpha) {
+            code += '#define SPECULAROVERALPHA 1\n';
```

Comment:
I don't think you need to define this as 1. I believe you can simply do: `#define SPECULAROVERALPHA`

Yeah, I see that `CLEARCOAT` is defined as 1 below, but I guess that's unnecessary too. :smile: 

cc @raytranuk

Reply:
yes - I will remove the 1 from CLEARCOAT too

Reply:
No problem. Will fix.